File: C Functions Task/todo2/biggest.c
#include <stdio.h>
void getBiggest(int a,int b)
{
  int result;
  result= a>b?a:b;
  printf("%d\n",result);
}
int main()
{
    getBiggest(256,38);
    getBiggest(30,34);
    return 0;
}

File: C Functions Task/biggest.java
public class Main{
public static void doBiggest(int a,int b){
    int result;
    result=a>b?a:b;
    System.out.printf("%d\n",result);
}
public static void main(String[]args){
    doBiggest(40,56);
    doBiggest(32,34);
}
}